lib/src/BUILD.bazel

```diff
@@ -14,18 +17,20 @@ cc_library(
         "subtree.c",
         "tree_cursor.c",
         "tree.c",
-    ] + glob(["*.h"]),
+    ] + glob(["**/*.h"]),
     deps = ["//lib/include/tree_sitter:hdrs"],
-    copts = [
-        "-x", "c",
-        # "-O3",
-        # "-Wno-unused-function"
-
-        "-Ilib/include",
-        "-Ilib/src"
-    ] + select({
-        "@platforms//os:linux": ["-std=gnu11"],
-        "//conditions:default": ["-std=c11"],
+    includes = [
```

Comment:
I don't think `includes` is necessary, is it? The headers are pulled in by the deps.

Reply:
that might be! I assumed it was necessary because of these lines: https://github.com/obazl/tree-sitter/pull/1/files#diff-27a9e9404f965a0b6bad96c700a2c74084cc3df477018a5ff48fa5dc9dd9727fL24-L25

Reply:
I went ahead and merged, and I've removed the `includes` in my working dir. Still seems to work ok. The thing about `includes` is that it propagates to things that depend on the target, which is not what we want.

I've added a test target.  See `examples/imp` int the test branch.